modules/install/pages/install.adoc

```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
+
+* Copy the zip file under `/opt`.
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# cp ~/Downloads/jboss-eap-8.0.0-installation-manager.zip /opt
+----
+
+* Swtich to /opt and uncompress the zip file:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost opt]# unzip /opt/jboss-eap-8.0.0-installation-manager.zip
+----
+
+* Switch to the uncompressed folder:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost opt]# cd jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin
+----
+
+* Install JBoss EAP by running the following command:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost bin]# ./jboss-eap-installation-manager.sh install --profile eap-8.0 --dir eap-8
+Installing profile: eap-8.0
+Using channels:
+# eap-8.0
+  manifest: org.jboss.eap.channels:eap-8.0
+_...output omitted..._
+----
++
+Review the end user license agreement, and follow the rest of the prompts to install JBoss EAP.
++
+[subs="+quotes,+macros"]
+----
+Accept the agreement(s) [y/N]y
+Feature-packs resolved.
+Packages installed.
+Downloaded artifacts.
+JBoss modules installed.
+_...output omitted..._
+----
+
+* Start the JBoss EAP using the start script present under `/opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8/bin/standalone.sh`
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8/bin/standalone.sh
+----
+
+* Navigate to the JBOSS console at http://localhost:8080
+
+[NOTE]
+--
+The jboss-eap-installation-manager is also supported on Microsoft Windows. To use this script on a Windows machine, replace the .sh with .bat in your script.
+--
 
 == Lab 2: Installing JBoss EAP using the JAR installer
 
-. Step 1
-. Step 2
-.. Step 2.a
-.. Step 2.b
-. Step 3
+* Install OpenJDK 17 if not already installed:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# sudo yum install java-17-openjdk -y
+----
+
+* Install the GUI if not already installed on RHEL 9:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# sudo yum install java-17-openjdk -y
+----
+
+* Create a user named jboss with no shell access:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# useradd -s /sbin/nologin jboss
+----
+
+* Download the JBoss EAP zip file from https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=appplatform&downloadType=distributions[`Red Hat JBoss Enterprise Application Platform 8.0 Installer`].
+
+* Open a terminal instance and navigate to the directory containing the downloaded JBoss EAP installer file.
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# cd Downloads/
+[root@localhost Downloads]# ls -l jboss-eap-8.0.0-installer.jar
+-rw-r--r--. 1 root root 28141889 Apr  1 09:35 jboss-eap-8.0.0-installer.jar
+----
+
+* Run the graphical installer using the following command:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost Downloads]# java -jar jboss-eap-8.0.0-installer.jar
```

Comment:
I think you need to launch the installer as the jboss user? check AD248. if installing as root, then you need to chmod the folders where you installed EAP under /opt